Marlin/src/gcode/feature/pause/M600.cpp

```diff
@@ -60,7 +60,7 @@ void GcodeSuite::M600() {
     int8_t DXC_ext = target_extruder;
     if (!parser.seen('T')) {  // If no tool index is specified, M600 was (probably) sent in response to filament runout.
                               // In this case, for duplicating modes set DXC_ext to the extruder that ran out.
-      #if ENABLED(FILAMENT_RUNOUT_SENSOR)
+      #if ENABLED(FILAMENT_RUNOUT_SENSOR) && NUM_RUNOUT_SENSORS>1
```

Comment:
There should be spaces around operators.

Please read http://marlinfw.org/docs/development/coding_standards.html asap.